scripts/create-kubestellar-demo-env.sh

```diff
@@ -58,10 +58,11 @@ echo -e "KubeStellar Version: ${kubestellar_version}"
 
 echo -e "Checking that pre-req softwares are installed..."
 if [ "$k8s_platform" == "kind" ]; then
-    curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/v${kubestellar_version}/hack/check_pre_req.sh | bash -s -- -V kflex ocm helm kubectl docker kind
-else
-    curl -s https://raw.githubusercontent.com/clubanderson/kubestellar/refs/heads/add-k3d-to-create-demo-env/hack/check_pre_req.sh | bash -s -- -V kflex ocm helm kubectl docker k3d
     # curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/v${kubestellar_version}/hack/check_pre_req.sh | bash -s -- -V kflex ocm helm kubectl docker kind
+    curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/refs/heads/main/hack/check_pre_req.sh | bash -s -- -V kflex ocm helm kubectl docker kind
```

Comment:
This one does not need to be changed, right? The released check_pre_req.sh is as good as the one in `main` for all but "k3d", right?

Reply:
updated